lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -31,6 +31,8 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/go-openapi/runtime/middleware"
 
+	"github.com/vmware/govmomi/vim25/types"
```

Comment:
Go imports nit - `github.com/vmware/govmomi/vim25/types` should be in the same group as the other `vmware` imports.

Reply:
I thought we made a distinction between vmware imports from this project and external vmware projects (e.g. govmomi) but alright i'll delete the newline beneath it so it's in the same group